packages/x-charts/src/BarChart/BarLabel/BarLabel.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { styled, useThemeProps } from '@mui/material/styles';
+
+import { animated } from '@react-spring/web';
+import { useSlotProps } from '@mui/base';
```

Comment:
Prefer more specific package imports whenever possible. 😉 
```suggestion
import { useSlotProps } from '@mui/base/utils';
```